vpc-endpoints.tf

```diff
@@ -1,3 +1,17 @@
+data "aws_iam_policy_document" "default" {
```

Comment:
we shouldn't need this, instead we can set the policy to `null` most likely. see below